### PR TITLE
Fix wordpress plugin tests

### PR DESCRIPTION
**Pull request by tortillaj**
This is just some cleanup of the Wordpress plugin. I added more tests to cover the plugin more thoroughly too. 
 
1. I fixed two bugs. Before this PR, the [flushCaches](https://github.com/ProboCI/probo/pull/95/files#diff-290329cbdb7390784f1b4e9bbd81125dR43) option could never be false. Also, the [Container.prototype.inspect](https://github.com/ProboCI/probo/pull/95/files#diff-599aa5690d4bc1ef4f297bfe52690761R259) method needs to be fully runnable as a promise. The AbstractPlugin calls this without `done` defined, and before this PR that causes an error.
1. Also, I fixed some small linting errors to get this passing code standards.
1. Lastly, I updated the WP plugin tests to make them pass. The last round of updates to it broke the tests.

### CODE REVIEW

File: lib/plugins/TaskRunner/WordPressApp.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ class WordPressApp extends LAMPApp {
 
     this.options.siteFolder = options.siteFolder || 'default';
     this.options.profileName = options.profileName || 'standard';
-    this.options.flushCaches = !!options.flushCaches || true;
+    this.options.flushCaches = String(options.flushCaches).toLowerCase() === 'true';
```
Comment (lliss) [inline lib/plugins/TaskRunner/WordPressApp.js]: I think this should default to `true`

File: package.json [hunk 2]
```diff
@@ -4,7 +4,10 @@
   "description": "The main repository for the probo project.",
   "homepage": "http://probo.ci",
   "main": "index.js",
-  "repository": "https://github.com/ProboCI/probo.git",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ProboCI/probo.git"
```
Comment (lliss) [inline package.json]: What is the purpose of this change?

Comment (tortillaj) [inline package.json]: npm just formatted it this way when I validated the settings in this file.

File: lib/plugins/TaskRunner/WordPressApp.js (hunk 1 above)
Comment (tortillaj) [inline lib/plugins/TaskRunner/WordPressApp.js]: @lliss so if the option is empty or not provided, it should be true?

File: package.json (hunk 2 above)
Comment (lliss) [inline package.json]: got it.

File: lib/plugins/TaskRunner/WordPressApp.js (hunk 1 above)
Comment (lliss) [inline lib/plugins/TaskRunner/WordPressApp.js]: I think so. I almost all cases you want the cache flushed when you set up a WP site on a different host. And we're doing all sorts of DB manipulation and URL rewriting so I can't imagine you'd wouldn't want the cache flushed. If you did, you'd probably have a very special reason for it.

Comment (tortillaj) [inline lib/plugins/TaskRunner/WordPressApp.js]: Ok, just making sure I understood what you wanted. Will post an update soon.

Comment (lliss) [inline lib/plugins/TaskRunner/WordPressApp.js]: Nah. My suggestion there doesn't work.

Comment (tortillaj) [inline lib/plugins/TaskRunner/WordPressApp.js]: @lliss ok good to go here ... yay javascript weak typing
